Small updates to the project so it builds after a fresh clone of the repo.

robbforce opened the pull request.

SanSYS commented:
> C:\Windows\System32\

System may be located in another disk, I recommend to use path like this: %WINDIR%\system32\

SanSYS commented on Bonobo.Git.Server/Bonobo.Git.Server.csproj:
```diff
@@ -654,9 +659,9 @@
 if not exist "$(TargetDir)NativeBinaries" md "$(TargetDir)NativeBinaries"
 if not exist "$(TargetDir)NativeBinaries\x86" md "$(TargetDir)NativeBinaries\x86"
-xcopy /s /y /d "$(SolutionDir)packages\LibGit2Sharp.0.18.1.0\lib\net40\NativeBinaries\x86\*.*" "$(TargetDir)NativeBinaries\x86"
+C:\Windows\System32\xcopy /s /y /d "$(SolutionDir)packages\LibGit2Sharp.0.18.1.0\lib\net40\NativeBinaries\x86\*.*" "$(TargetDir)NativeBinaries\x86"
```
> C:\Windows\System32\

System may be located in another disk, I recommend to use path like this: %WINDIR%\system32\

if it's possible, of course :)

SanSYS replied:
> C:\Windows\System32\

System may be located in another disk, I recommend to use path like this: %WINDIR%\system32\

if it's possible, of course :)

padremortius commented:
Environment variable **PATH** always contains the path %WINDIR%\system32\

jakubgarfield commented:
Please update the xcopy as per SanSYS suggestion before merging.

jakubgarfield commented:
So there is no need to prefix?

padremortius commented:
Full path to xcopy not need.

robbforce commented:
Like I mentioned in the commit, visual studio on my system (x64) exits before it copies the files, until I added the path to the xcopy command. I've updated the path with the %windir% variable now.
